CSharp/DevVmPowershell/DevVmPsModules/AddApplicationFromApplicationLibraryModule.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+using Helpers;
+
+namespace DevVmPsModules
+{
+	[Cmdlet(VerbsCommon.Add, "ApplicationFromApplicationLibrary")]
+	public class AddApplicationFromApplicationLibraryModule : BaseModule
+	{
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 0,
+			HelpMessage = "Name of the Relativity Instance")]
+		public string RelativityInstanceName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 1,
+			HelpMessage = "Username of the Relativity Admin")]
+		public string RelativityAdminUserName { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 2,
+			HelpMessage = "Password of the Relativity Admin")]
+		public string RelativityAdminPassword { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 3,
+			HelpMessage = "Workspace Artifact Id that you want to install the application in")]
+		public int WorkspaceId { get; set; }
+
+		[Parameter(
+			Mandatory = true,
+			ValueFromPipelineByPropertyName = true,
+			ValueFromPipeline = true,
+			Position = 4,
+			HelpMessage = "The Guid of the Application you want to install")]
+		public string ApplicationGuid { get; set; }
+
+		protected override void ProcessRecordCode()
+		{
+			//Validate Input arguments
+			ValidateInputArguments();
+
+			IConnectionHelper connectionHelper = new ConnectionHelper(RelativityInstanceName, RelativityAdminUserName, RelativityAdminPassword);
+			IApplicationInstallHelper applicationInstallHelper = new ApplicationInstallHelper(connectionHelper);
```

Comment:
Isn't there already an IApplicationInstallHelper in the API? How are we able to create an interface with the same name?

Reply:
@calimetikcura There is IApplicationInstallManager in the API